src/Crypto/RNCryptor/Types.hs

```diff
@@ -70,9 +73,21 @@ randomSaltIO :: Int -> IO ByteString
 randomSaltIO sz = C8.pack <$> forM [1 .. sz] (const $ randomRIO ('\NUL', '\255'))
 
 --------------------------------------------------------------------------------
+makeKey :: ByteString -> ByteString -> ByteString
+makeKey = generate (prfHMAC (undefined::SHA1)) (Parameters 10000 32) -- userKey salt
```

Comment:
Why use `undefined` here? It seems suspicious. From the signature of `prfHMAC`, you can pass this `SHA1` datatype as input, as expected:

https://hackage.haskell.org/package/cryptonite-0.19/docs/Crypto-Hash-Algorithms.html#t:SHA1

Any reason why not doing that?